src/lua/api-gateway/validation/validatorsHandler.lua

```diff
@@ -241,7 +241,7 @@ end
 function ValidatorsHandler:saveContextInRequestVars(localContext)
      for k,v in pairs(localContext) do
     -- for i,k in pairs(varsToSet) do
-        if ngx.var[k] ~= nil and type(localContext[k]) == "string" then
+        if ngx.var[k] ~= nil and (type(localContext[k]) == "string" or type(localContext[k]) == "number") then
```

Comment:
You could use v instead of localContext[k]

Reply:
@andralungu , @atrifan, @bogdanciuca  WDYT if we also check for boolean?

Reply:
@corinapurcarea Are we setting any booleans on `ctx`?

Reply:
planning ahead

Reply:
@corinapurcarea it's ok